examples/google-cloud-speech/output/google-cloud-speech/lib/google/cloud/speech/v1/cloud_speech_services_pb.rb

```diff
@@ -0,0 +1,56 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
This file is generated by a plugin that we do not plan to use. We'll likely generate something very similar, but do we want to change it? 

For example, we could put it in the client instead of having a separate file and/or add/change the comments to clarify that it's the low-level api and not meant for end users.